deploy/dashboard.yaml

```
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dashboard
+  namespace: radius-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dashboard
+  template:
+    metadata:
+      labels:
+        app: dashboard
+    spec:
+      serviceAccountName: dashboard-account
+      containers:
+        - name: dashboard
+          image: ghcr.io/aaroncrawfis/dashboard:latest
```

Is this just for testing?

---

Ah yeah need to change that back, thanks for the catch!
```suggestion
          image: ghcr.io/radius-project/dashboard:latest
```